tests/sentry/api/endpoints/test_organization_metrics_query.py

```diff
@@ -106,3 +106,20 @@ def test_query_with_disabled_org(self):
                     "includeSeries": "false",
                 },
             )
+
+    def test_recursion_error_query(self):
+        conds = " OR ".join([f'transaction:"{e}"' for e in range(500)])
```

Comment:
shouldn't this test be *in snuba-sdk* ?

Reply:
This is tested more exhaustively in the snuba-sdk. This is just an end-to-end test to make sure that the change actually does not introduce any breakage of the API.

Reply:
personally would prefer we leave the test in snuba-sdk rather than re-testing it here